### PR TITLE
Fixed class B multicast handling in LoRaMacClassBProcessMulticastSlot()

### PR DESCRIPTION
**Pull request by MarekNovakACRIOS**
- if a channel is not enabled, do not compute ping offset in ComputePingOffset(), I was getting a situation with "modulo zero" leading to a hard fault with a *disabled* channel being passed to ComputePingOffset() in LoRaMacClassBProcessMulticastSlot().
- do not use hard-coded constant of 4 instead of LORAMAC_MAX_MC_CTX macro - if this macro is changed to for example 2, then a hard fault or out of bound array access situation can happen.

### CODE REVIEW

File: src/mac/LoRaMacClassB.c
Comment (mluis1) [inline src/mac/LoRaMacClassB.c:1138]: Please add a space after `(` and before `)`.

File: src/mac/LoRaMacClassB.c
Comment (mluis1) [inline src/mac/LoRaMacClassB.c:1157]: Please add a space after `(` and before `)`.

**Review by mluis1 — CHANGES_REQUESTED** [review]
Thanks for the proposition.

Before merging could you please push the fix for the 2 comments that I have.

**Comment by MarekNovakACRIOS** [comment]
I have added the spaces, so it should be OK from coding style perspective.

**Review by mluis1 — APPROVED** [review]

**Pull request merged**
